validator/main.py

```diff
@@ -178,21 +246,31 @@ def set_callable(self, llm_callable: Union[str, Callable, None]) -> None:
                     "Check out ProvenanceV1 documentation for an example."
                 )
 
-            def openai_callable(text: str, topics: List[str]) -> str:
+            def openai_callable(text: str) -> str:
                 api_key, api_base = self.get_client_args()
                 response = OpenAIClient(api_key, api_base).create_chat_completion(
                     model=llm_callable,
+                    response_format={"type": "json_object"},
```

Comment:
Instead of passing self._json_schema in the prompt, can you use it via function calling?

E.g. as shown here -> https://cookbook.openai.com/examples/how_to_call_functions_with_chat_models

Reply:
done